docs/developer/index.rst

```diff
@@ -98,6 +98,31 @@ run allows us to generate a coverage report after that::
 The report is in ``report/index.html``, which links to individual
 files and shows exactly which lines were excuted while running the tests.
 
+Run doctests locally
```

Comment:
A question for this section. You show runs for four explicit files. Are these the only files that have been converted, or just an example set?